### ISSUE TITLE
AWS::ApiGatewayV2::Api 'IpAddressType' not supported

### ISSUE THREAD

**Issue by nosnilmot** [aws-cloudformation/cfn-lint#4058]
### CloudFormation Lint Version

cfn-lint 1.32.1 / git eecec889d

### What operating system are you using?

Mac

### Describe the bug

`cfn-lint` does not recognize the (new) `IpAddressType` property for `AWS::ApiGatewayV2::Api` resources

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-apigatewayv2-api.html#cfn-apigatewayv2-api-ipaddresstype

```
$ cfn-lint apigateway.yaml
E3002 Additional properties are not allowed ('IpAddressType' was unexpected)
apigateway.yaml:8:7
```

### Expected behavior

cfn-lint passes

### Reproduction template

```yaml
AWSTemplateFormatVersion: '2010-09-09'
Resources:
  ApiGateway:
    Type: AWS::ApiGatewayV2::Api
    Properties:
      ProtocolType: HTTP
      IpAddressType: dualstack
```

**Comment by nosnilmot** [comment]
running `cfn-lint --update-specs` fixes this, sorry for the noise

**Comment by kddejong** [comment]
It was covered in our automated jobs - https://github.com/aws-cloudformation/cfn-lint/pull/4052/.  The next release will cover this in the base install.

**Comment by nosnilmot** [comment]
@kddejong my colleague has identified that this still does not work for regions other than `us-east-1`:

```
$ cfn-lint --version
cfn-lint 1.33.2
$ cfn-lint --update-specs
$ cfn-lint --regions eu-west-1 -- apigateway.yaml 
E3002 Additional properties are not allowed ('IpAddressType' was unexpected)
apigateway.yaml:7:7
$ cfn-lint --regions us-east-1 -- apigateway.yaml 
$
```

AWS support ticket 174472019500232 is opened to address mismatch in regional resource schemas, but maybe you have some influence there too? 😄
